### ISSUE TITLE
PuffingBilly fails to rescue when webkit not present and raises 'cannot load such file'

### ISSUE THREAD

**Issue by ahharvey** [oesmith/puffing-billy#169]
I have added Puffing Billy to a Rails app as per the instructions in the readme. Running the test suite raises:

`in 'require': cannot load such file -- capybara/webkit (LoadError)`

from 

`/lib/billy/browsers/capybara.rb:15`

I am using Poltergeist rather than Webkit, and so don't need this file loaded. I was therefore surprised that this line raises rather than rescuing from an error. 

I asked about this on StackOverflow and received the following 

> In general it's a bad idea to use an inline rescue in Ruby. Inline rescue and a "unfiltered" rescue (without specifying the type of error you want caught) both catch any exceptions that inherit from `StandardError`. `LoadError` is not a descendant of `StandardError` so it's is not caught by this inline rescue.

and a suggestion to modify `/lib/billy/browsers/capybara.rb` as follows:

```
DRIVERS.each do |name, driver|
  begin
    require driver
    send("register_#{name}_driver")
  rescue LoadError
    # This skips to the next driver
    # It would be nice to add logging here to notify that the driver was skipped
  end
end
```

Is this a genuine bug within Puffing-Billy, or is something else going on? I'm happy to submit a PR if needed.

**Comment by ronwsmith** [comment]
Likely a real issue.  Big change was merged yesterday that changed this.

[Here's the old code](https://github.com/oesmith/puffing-billy/pull/158/files#diff-2666ec19a9cf184d520c8a465e40bed4), which is similar to yours in how it rescues.

I would skip the logging though as it will be consistently noisy for most people.

**Comment by ahharvey** [comment]
I've submitted a very small PR which reverts to the old code. #170

**Comment by ronwsmith** [comment]
Fixed in 0.9.1
